docs/validate_switch_config.md

```diff
@@ -73,10 +73,6 @@ Generated config file
 Output results to a file
 
 
```

Comment:
Isn't the new --custom-config missing here?

Reply:
Custom config won't be in validate.  We will be validating against generated config with the custom config already applied.